flocker/node/agents/gce.py

```diff
@@ -822,6 +724,249 @@ def _stop_node(self, node_id):
 
         :param unicode node_id: The compute_instance_id of the node to stop.
         """
+        self._volume_manager.stop_node(node_id)
+
+
+class IGCEVolumeManager(Interface):
```

Comment:
:question: We call this a GCEVolumeManager but it has operations in the interface that operate on nodes.  Is this named correctly?